doc/source/conf.py

```diff
@@ -227,10 +237,11 @@
 #latex_font_size = '10pt'
 
 # Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title, author, documentclass [howto/manual]).
+# (source start file, target name, title, author,
+#  documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'nibabel.tex', u'NiBabel Documentation',
-   u'NiBabel Authors', 'manual'),
+    ('index', 'nibabel.tex', u'NiBabel Documentation',
+     u'NiBabel Authors', 'manual'),
 ]
```

Comment:
Not sure if this is done in the rest of the code (it is at the places I checked briefly now), but for multi-line lists & dicts, I usually put the close brace on the same line as the last element:
```python
     u'NiBabel Authors', 'manual'),]
```